packages/styles/src/components/choice-control-card.scss

```diff
@@ -0,0 +1,133 @@
+@use '../variables/color';
+@use '../variables/commons';
+@use '../variables/spacing';
+
+.radio-button-card,
+.checkbox-button-card {
+  --post-card-select--hover-bg: #{color.$gray-10};
+
+  position: relative;
+  display: flex;
+  gap: 0 spacing.$size-mini;
+
+  width: 100%;
+
+  color: color.$gray-80;
+  background-color: color.$white;
+  border: 2px solid color.$gray-60;
+  border-radius: commons.$border-radius;
+  padding: spacing.$size-regular;
+  transition: color 100ms ease-in-out, background-color 100ms ease-in-out,
+    border-color 100ms ease-in-out;
+
+  // Checked
+  &:where(:has(input:checked)) {
```

Comment:
:has is not supported by Firefox at the moment.

Reply:
Added a fallback with classes until it is.